src/fest.ml

```diff
@@ -1,27 +1,32 @@
 external test : string -> (unit -> unit) -> unit = "test"
 [@@mel.module "node:test"]
+(** Create a test with a given name and callback function that runs the test *)
 
 module Promise = struct
   let ( let* ) p f = Js.Promise.then_ f p
 
-  (* The promise-based version is needed for nested tests,
-     see https://nodejs.org/api/test.html#subtests *)
   external test : string -> (unit -> unit Js.Promise.t) -> unit Js.Promise.t
     = "test"
   [@@mel.module "node:test"]
+  (** Create a test with a given name and callback function that runs the test and returns a promise.
+
+  The promise-based version is needed for nested tests, see {: https://nodejs.org/api/test.html#subtests } *)
```

Comment:
Should this be a comment at module level? Rather than function level here. (iow, should we move it to line 4)